File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        
     int ans=0;
     int num=0;
     for(int i=0;i<nums.size();i++)
     {
        
        if(nums[i]==1)
        {
          num++;
          ans=max(ans,num);
        }
        if(nums[i]!=1)
        {
            num=0;

        }
     }
     return ans;


    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let num=0;
    let count=0;
    for(let x of nums)
    {
        if(x==1)
        {
            num++;
            // count=max(count,num);
            if(num>count)
            {
                count=num;
            }
        }
        else
        {
            num=0;
        }
    }
    return count;
    
};